packages/types/src/clerk.ts

```diff
@@ -990,7 +990,7 @@ export type SignInCombinedProps = RoutingOptions & {
   LegacyRedirectProps &
   AfterSignOutUrl;
 
-interface TransferableOption {
+export interface TransferableOption {
```

Comment:
I'm getting a transitive type dependency error because the `SignInProps` is exported but not the `TransferableOption` interface that it depends on